elasticache/redis/README.md

```diff
@@ -2,20 +2,65 @@
 
 Run an Elasticache Redis cluster as an Acorn with a single click or command.
 
+## Usage
+
+From the CLI you can run the following command to create an Elasticache Redis cluster.
+
+```shell
+acorn run -n elasticache-redis-cluster ghcr.io/acorn-io/elasticache/redis:v0.#.#
+```
+
+From an Acornfile you can create the cluster by using the acorn too.
```

Comment:
Should 'acorn' be capitalized? Should it be qualified with "Redis"?

Reply:
I don't know if it should be capitalized. I was capitalizing it before but it feels wrong because "acorn" in this sense isn't a proper noun.

Reply:
I've qualified it with "Redis" in the latest commit.